src/c-sample.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022, Jolla Ltd.
```

Comment:
```suggestion
 * Copyright (c) 2022 Jolla Ltd.
```
Official syntax is without comma

Reply:
I literally copypasted this from the example from the official document, it has a comma